lib/main.go

```diff
@@ -200,6 +200,41 @@ func (obj *Main) Exit(err error) {
 	obj.exit <- err // trigger an exit!
 }
 
+// logic to create (tmp) prefix directory
```

Comment:
comment should be in standard `golint` format -- full sentences (ends with period), and starts with exact function name. The linter doesn't enforce on private functions, but it's a good habit to have.